x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -416,13 +410,53 @@ const updateExistingIndex = async ({
     // if update fails, rollover data stream
   } catch (err) {
     try {
+      // get the data_stream values to compose datastream name
+      const searchDataStreamFieldsResponse = await callCluster('search', {
+        index: indexTemplate.index_patterns[0],
+        body: {
+          size: 1,
+          query: {
+            bool: {
+              must: [
```

Comment:
nit: I think we could put this in the `filter` context so elasticsearch doesn't have to calculate the score right?